.npmignore

```diff
@@ -4,3 +4,7 @@
 /.npmignore
 /.travis.yml
 /npm-debug.log
+/email.bundle.js
```

Comment:
what do you think about moving these to a "rollup/" folder? or a "build" or "dist" folder?

Reply:
moved to `rollup/`